custom/icds_reports/tasks.py

```diff
@@ -545,13 +545,13 @@ def _run_custom_sql_script(commands, day=None, db_alias=None):
 @track_time
 def aggregate_awc_daily(day):
 
-    agg_daily_dates = [force_to_date(day) - timedelta(days=2),
-                       force_to_date(day) - timedelta(days=1),
-                       force_to_date(day)]
+    agg_daily_dates = [(force_to_date(day) - timedelta(days=2), False),
```

Comment:
to address (1)

Reply:
nit: this would be more readable as dicts and kwargs since a boolean like this gives you very little context as to what it represents.